admin/tests/unit/adapters/certification-center-membership-test.js

```diff
@@ -20,4 +23,34 @@ module('Unit | Adapter | certificationCenterMembership', function(hooks) {
       assert.equal(query.filter.certificationCenterId, undefined);
     });
   });
+
+  module('#createRecord', function() {
+
+    module('when createByEmail is true', function() {
+
+      test('should call certification-centers', async function(assert) {
```

Comment:
**Suggestion**: should call certification-centers/${certificationCenterId}/certification-center-memberships

Reply:
Done